llpc/translator/lib/SPIRV/SPIRVReader.cpp

```diff
@@ -115,6 +115,10 @@ cl::opt<Vkgc::DenormalMode> Fp32DenormalModeOpt(
                clEnumValN(Vkgc::DenormalMode::FlushToZero, "ftz", "Denormal input/output flushed to zero"),
                clEnumValN(Vkgc::DenormalMode::Preserve, "preserve", "Denormal input/output preserved")));
 
+cl::opt<bool> SpirvOverrideWorkaroundStorageImageFormats("spirv-override-workaround-storage-image-formats",
+                                                         cl::desc("Override the workaroundStorageImageFormats setting"),
```

Comment:
Maybe add a more descriptive text here? Something that describes the impact of the setting.

Reply:
Done